manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -357,14 +368,15 @@ spec:
             created:
               type: boolean
           type: object
-  version: v1beta1
+      type: object
+  version: v1alpha1
```

Comment:
@maxsmythe @julianKatz do we want `version` to be `v1alpha1` or `v1beta1`?

Reply:
I think it should be `v1beta1` here https://github.com/open-policy-agent/frameworks/blob/3f51770b2910689616b1c78b7b5e75010cd899a0/constraint/config/crds/templates.gatekeeper.sh_constrainttemplates.yaml#L109

Reply:
version needs to be equal to the first item in the `versions` list. It looks like it doesn't actually do anything except break if it doesn't match:

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#customresourcedefinitionspec-v1beta1-apiextensions-k8s-io

Reply:
> Note: In apiextensions.k8s.io/v1beta1, there was a version field instead of versions. The version field is deprecated and optional, but if it is not empty, it must match the first item in the versions field.

Looks like `version` is deprecated. Do we want to keep it?

Reply:
It's okay to remove but @julianKatz wasn't able to find a way for kubebuilder to generate without `version` I believe

Reply:
Yeah, it should go away when we switch to generating v1 CRDs (which don't have this field)